app/views/calculators/guaranteed_income/_form.html.erb

```diff
@@ -0,0 +1,38 @@
+<%
+  form = local_assigns[:form]
+  errors = form&.errors
+%>
+
+<h2 id="calculator">Estimate how much your guaranteed income could be</h2>
+
+<%= render 'calculators/error_summary', messages: errors&.messages if form&.invalid? %>
+
+<form action="/guaranteed-income/estimate#form" method="get">
+  <%= render 'calculators/form_input_currency',
+             field: :pot,
+             value: form&.pot,
+             label: 'Amount you\'ll have your pot when you retire',
```

Comment:
This is all placeholder copy.
On 23 Feb 2016 3:36 p.m., "Andrew Garner" <notifications@github.com> wrote:

> In app/views/calculators/guaranteed_income/_form.html.erb
> <https://github.com/guidance-guarantee-programme/pension_guidance/pull/471#discussion_r53796669>
> :
>
> > @@ -0,0 +1,38 @@
> > +<%
> > +  form = local_assigns[:form]
> > +  errors = form&.errors
> > +%>
> > +
> > +<h2 id="calculator">Estimate how much your guaranteed income could be</h2>
> > +
> > +<%= render 'calculators/error_summary', messages: errors&.messages if form&.invalid? %>
> > +
> > +<form action="/guaranteed-income/estimate#form" method="get">
> > +  <%= render 'calculators/form_input_currency',
> > +             field: :pot,
> > +             value: form&.pot,
> > +             label: 'Amount you\'ll have your pot when you retire',
>
> 'Amount you’ll have your pot when you retire'
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/guidance-guarantee-programme/pension_guidance/pull/471/files#r53796669>
> .
>